app/models/batch_context.rb

```diff
@@ -23,7 +23,9 @@ class BatchContext < ApplicationRecord
     'file' => 3,
     'media' => 4,
     '3d' => 5,
-    'document' => 6
+    'document' => 6,
+    'maps' => 7,
```

Comment:
Note, it has to be maps because map is a reserved keyword for enums:

```You tried to define an enum named "content_structure" on the model "BatchContext", but this will generate a class method "map", which is already defined by ActiveRecord::Relation.```

Reply:
Where is it trying to eval the key?

Reply:
Seems to happen when the class is required, the tests don't even run if you use a key of `map`:

```
$ be rspec

An error occurred while loading ./spec/models/batch_context_spec.rb.
Failure/Error:
  enum content_structure: {
    'simple_image' => 0,
    'simple_book' => 1,
    'book_as_image' => 2, # Deprecated
    'file' => 3,
    'media' => 4,
    '3d' => 5,
    'document' => 6,
    'map' => 7,
    'webarchive_seed' => 8,

ArgumentError:
  You tried to define an enum named "content_structure" on the model "BatchContext", but this will generate a class method "map", which is already defined by ActiveRecord::Relation.
# ./app/models/batch_context.rb:19:in `<class:BatchContext>'
# ./app/models/batch_context.rb:3:in `<top (required)>'
# /Users/petucket/.rvm/gems/ruby-2.7.1/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:26:in `require'
# /Users/petucket/.rvm/gems/ruby-2.7.1/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:26:in `require'
# ./spec/models/batch_context_spec.rb:3:in `<top (required)>'
```

Reply:
Ah, because it's the ActiveRecord enum method.